app/main.js

```diff
@@ -0,0 +1,70 @@
+{
+    'use strict';
+
+    const electron = require('electron');
+    const app = electron.app;
+    const BrowserWindow = electron.BrowserWindow;
+
+    // simple parameters initialization
+    let electronConfig = {
+        "URL_LAUNCHER_TOUCH": (process.env.URL_LAUNCHER_TOUCH == null) ? 0 : parseInt(process.env.URL_LAUNCHER_TOUCH),
```

Comment:
Yes, even if the REDME clearly specifies the legal values, i can see this happening a lot, and it's a good practice anyways